### PR TITLE
Add status support to remove upstart scripts

### PR DESCRIPTION
**Pull request by ilyapoz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Tunables
 * `upstart_log_root` (string) - Directory for logs
 * `upstart_log_path` (string) - Path for log file
 * `upstart_capture_errors` (boolean) - Redirect STDERR to STDOUT?
+* `state` (present|absent) - Install or remove init script
```
Comment (aaronpederson) [inline README.md]: We prefer the naming be more explicit `upstart_init_state` 
Boolean would also be preferred rather than enumeration.

**Comment by stevenharradine** [comment]
Can you explain your use case for this functionality?  We try not to account for all the possible states that can arise from changes over time.  We start all systems from a "clean" state so if you were to want to remove something in a new system it would never be placed on the system in the first place.  If you feel we are incorrect in this line of thought please let us know.

**Comment by ilyapoz** [comment]
I do feel that this line of thought is a bit simplified compared to the ansible philosophy. In ansible we describe the state the system must be found in, not just the steps to get there from some known starting position (i.e. a 'clean' system). Being able to get a clean system is a luxury not every setup has. So a simple use case is changing the configuration, just removing old upstart scripts.
Te be honest, this pull request I used as a learning tool for ansible as well as test kitchen, I since moved away from this custom upstart script generation to `foreman`. But I could return back to it in some other project, I see value in this role.

**Comment by aaronpederson** [comment]
Team Approved, @ilyapoz would you be able to fix the merge conflict and we will merge it in asap.

**Pull request merged**